src/Poseidon/Janno.hs

```diff
@@ -60,7 +73,7 @@ import qualified Data.Text                            as T
 import qualified Data.Text.Encoding                   as T
 import qualified Data.Vector                          as V
 import           GHC.Generics                         (Generic)
-import           Network.URI                          (isURI)
+import           Network.URI                          (isURIReference)
```

Comment:
What? They changed the function name just like that? Was there a version change?

Reply:
Nono - I changed that. Somehow the more specific `isURI` doesn't recognize the `ftp.sra.ebi.ac.uk/...` URLs in the test .ssf file. I had to switch to the more generic isURIReference

> isURI :: [String](https://hackage.haskell.org/package/base-4.16.3.0/docs/Data-String.html#t:String) -> [Bool](https://hackage.haskell.org/package/base-4.16.3.0/docs/Data-Bool.html#t:Bool)
> 
> Test if string contains a valid URI (an absolute URI with optional fragment identifier).
> 
> isURIReference :: [String](https://hackage.haskell.org/package/base-4.16.3.0/docs/Data-String.html#t:String) -> [Bool](https://hackage.haskell.org/package/base-4.16.3.0/docs/Data-Bool.html#t:Bool)
>
> Test if string contains a valid URI reference (an absolute or relative URI with optional fragment identifier).

https://hackage.haskell.org/package/network-uri-2.6.4.2/docs/Network-URI.html#g:3

Reply:
OK